### PR TITLE
tests: external_flash: sysbuild: Enable empty netcore

### PR DESCRIPTION
**Pull request by gchwier**
Updated sysbuild.conf to enable empty netcore core image.

### CODE REVIEW

File: tests/modules/mcuboot/external_flash/sysbuild.conf [hunk 1]
```diff
@@ -5,3 +5,9 @@
 #
 
 SB_CONFIG_BOOTLOADER_MCUBOOT=y
+SB_CONFIG_NETCORE_EMPTY=y
```
Comment (nordicjm) [inline tests/modules/mcuboot/external_flash/sysbuild.conf:8]: the netcore options would go in Kconfig.sysbuild, and only be enabled for nrf5340dk, not nrf52840dk

**Review by nordicjm — CHANGES_REQUESTED** [review]

**Review by sigvartmh — APPROVED** [review]
I saw that we filter for boards is there a specific reason for using boards over SOC? I'm just assuming that what you wanted to hit was the CPUAPP with these configs.

If there is a good reason I'm Ok with these changes.

File: tests/modules/mcuboot/external_flash/Kconfig.sysbuild [hunk 2]
```diff
@@ -7,4 +7,18 @@
...
+
+choice NETCORE
+	default NETCORE_EMPTY if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_THINGY53_NRF5340_CPU
+endchoice
+
+config SECURE_BOOT_NETCORE
+	default y if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_THINGY53_NRF5340_CPUAPP
+
+config NETCORE_APP_UPDATE
+	default y if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_THINGY53_NRF5340_CPUAPP
```
Comment (sigvartmh) [inline tests/modules/mcuboot/external_flash/Kconfig.sysbuild:22]: Here I think you could just use 1 option which is `SOC_NRF5340_CPUAPP` I would assume this satisfies your requirement to enable these KConfigs.

Comment (sigvartmh) [reply tests/modules/mcuboot/external_flash/Kconfig.sysbuild:22]: That being said the options you chose better reflects what are the supported boards.

**Comment by gchwier** [comment]
Thank you @sigvartmh . Updated with `SOC_NRF5340_CPUAPP` and rebased.

File: tests/modules/mcuboot/external_flash/Kconfig.sysbuild [hunk 3]
```diff
@@ -5,6 +5,20 @@
 #
 
 config PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY
-	default y if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF52840DK_NRF52840
+	default y if SOC_NRF5340_CPUAPP
```
Comment (nordicjm) [inline tests/modules/mcuboot/external_flash/Kconfig.sysbuild:8]: this is removing support for nrf52840dk?

Comment (gchwier) [reply tests/modules/mcuboot/external_flash/Kconfig.sysbuild]: right, to much copy-paste. Thank you!

**Review by nordicjm — APPROVED** [review]

**Pull request merged**